src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.BidirectionalSymbolSet.cs

```diff
@@ -47,15 +47,13 @@ public override ImmutableArray<ISymbol> GetAllSymbols()
         public override async Task InheritanceCascadeAsync(Project project, CancellationToken cancellationToken)
         {
             // Start searching using the current set of symbols built up so far.
-            var workQueue = new Stack<ISymbol>();
-            workQueue.Push(_allSymbols);
+            using var _ = ArrayBuilder<ISymbol>.GetInstance(out var workQueue);
+            workQueue.AddRange(_allSymbols);
```

Comment:
note: AddRange is fine here as _allSymbols is a set, so there's no concept of any sort of order to preserve when pushing onto the stack.  Other AddRange calls in this PR work similarly.